game/templates/game/board.html

```
@@ -660,6 +632,74 @@
+                        <span class="theme-swatch pastel-swatch" title="Pastel"></span>
+                        <span class="theme-swatch-name">Pastel</span>
+                    </label>
+                </div>
+            </div>
+
+            <div class="theme-modal-section">
+                <h3 class="section-title">Preferences</h3>
+                <div class="pref-switch-row">
+                    <span class="pref-label">🔊 Sound Enabled</span>
+                    <label class="switch-toggle">
+                        <input type="checkbox" id="modalSoundToggle">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)